recipes-connectivity/bluez5/bluez5/brcm43438.service

```diff
@@ -2,10 +2,12 @@
 Description=Broadcom BCM43438 bluetooth HCI
 ConditionPathIsDirectory=/proc/device-tree/soc/gpio@7e200000/bt_pins
 Before=bluetooth.service
+Wants=dev-serial1.device
```

Comment:
perhaps this should be
```
BindsTo=
```

Reply:
@kraj Why do you recommend `BindsTo`?

Reply:
to express correct deps

https://www.freedesktop.org/software/systemd/man/systemd.unit.html#BindsTo=

Reply:
Sure, but how does it help? The bind of usually makes a difference when you want to stop a service if it's dependency stops. But there is nothing to stop as this is just a script. I'm not sure how this applies in this case.

Reply:
I went with it for primarily because it really is a strong dependency: if the device goes away (which might happen), than the HCI configuration isn't valid anymore.  As I read `BindsTo` it represents this and would reflect it in the systemd state.  I don't think `Wants` wouldn't even stop the unit from running if the device wasn't there.